### ISSUE TITLE
LoRA fine-tuning hyper-parameters

### ISSUE THREAD

**Issue by ryusaeba** [locuslab/wanda#8]
It's really awesome study. May I have LoRA hyper-parameter and configuration to reproduce the results Section 5.2 describes?

**Comment by Eric-mingjie** [comment]
Thanks a lot for your interest in our study. Here we mostly used the same hyper-parameter setting as [alpaca-lora](https://github.com/tloen/alpaca-lora). Learning rate is set to be 1e-4, which was the main hyper-parameter we tuned.

**Comment by ryusaeba** [comment]
Nice :) We will give it a try. It would be much better if you share few mode code snippet.

**Comment by Eric-mingjie** [comment]
Yeah, i can share the code to you via email if you are interested (probably in a few days). It may be a little hacky but i can give you a description outlining the code.

**Comment by ryusaeba** [comment]
You are so nice! [Here ](yuming0727@gmail.com)is my mail.

**Comment by niksarrow** [comment]
Hi @Eric-mingjie 
Can you please share the code for LoRA fine tuning with me as well? 
niksarrow196@gmail.com
Thanks 🙂

**Comment by kiucho** [comment]
@Eric-mingjie 
If you have any fine-tuning code using LoRA, I'd love to hear about it.
If it's possible, please share it with me as well.
hoho5702@jbnu.ac.kr
Thank you.

**Comment by zyxxmu** [comment]
Hi @Eric-mingjie ,

Could you please share the code for LoRA fine-tuning with me (yuxinzhang@stu.xmu.edu.cn) as well?

Thank you very much!
